vite.embed.config.js

```diff
@@ -16,7 +18,19 @@ export default defineConfig(({ mode}) => {
       react(),
       tailwindcss(),
       ...(env.VITE_ENV === 'production' ? [removeAttrs(['data-cy'])] : []),
-      injectConfig(projectName)
+      injectConfig(projectName),
+      {
```

Comment:
Please move the plugin to a separate file like "injectConfig" and just call it in the vite config. The reason is that we need that plugin also in vite.config.js file which builds the standalone bundle. As we deliver both, I didn't find any better way to handle this double-build as having two vite configs. I know there are duplicates in both configs, this is still pending to resolve.